inst/doc/geoknife.R

```diff
@@ -4,6 +4,9 @@ options(continue=" ")
 options(width=60)
 library(knitr)
 library(geoknife)
+query <- geoknife::query
+`values<-` <- geoknife::`values<-`
+id <- geoknife::id
```

Comment:
that is odd. i agree that vignettes should be built in separate environments and am surprised that happens. now i see why you were thinking of hiding it from the user - it shouldn't actually apply to them if they're following the vignettes (in separate environments) themselves. yeah, maybe i like your current solution best...or load the conflicting libraries before geoknife in plot_geotiff.Rmd so that the re-masking doesn't even need to happen. 

could possibly put notes in the help files for `values<-`, `query`, and `id` about how to deal with masking issues (i.e., by using `::`) and known packages that mask these functions. then a user who runs into this issue on their own will have somewhere to turn.